deployments/admission-controllers/scheduler/templates/server.yaml.template

```diff
@@ -15,6 +15,7 @@ spec:
       labels:
         app: yunikorn
     spec:
+      imagePullSecrets:
```

Comment:
hi @ayubpathan 

When `imagePullSecrets` is empty, this will leave a field like the following:

```
spec:
      imagePullSecrets:
```

will this break the helm install? As it might expecting something for this field. Could you please verify this?

Reply:
The deployment wont break. The empty spec vars are ignored!